x-pack/test/functional/apps/transform/starting.ts

```diff
@@ -0,0 +1,97 @@
+        mode: 'batch',
+      },
+      {
+        suiteTitle: 'continuous transform with pivot configuration',
+        originalConfig: getPivotTransformConfig(PREFIX, true),
+        mode: 'continuous',
+      },
+      {
+        suiteTitle: 'batch transform with latest configuration',
+        originalConfig: getLatestTransformConfig(PREFIX, false),
+        mode: 'batch',
+      },
+      {
+        suiteTitle: 'batch transform with latest configuration',
+        originalConfig: getLatestTransformConfig(PREFIX, true),
+        mode: 'continuous',
+      },
+    ];
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/ecommerce');
+      await transform.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+
+      for (const testData of testDataList) {
+        await transform.api.createTransform(testData.originalConfig.id, testData.originalConfig);
+      }
+      await transform.testResources.setKibanaTimeZoneToUTC();
+      await transform.securityUI.loginAsTransformPowerUser();
+    });
+
+    after(async () => {
+      for (const testData of testDataList) {
+        await transform.testResources.deleteIndexPatternByTitle(testData.originalConfig.dest.index);
+        await transform.api.deleteIndices(testData.originalConfig.dest.index);
+      }
+
+      await transform.api.cleanTransformIndices();
+    });
+
+    for (const testData of testDataList) {
+      const transformId = testData.originalConfig.id;
+
+      describe(`${testData.suiteTitle}`, function () {
+        it('start transform', async () => {
+          await transform.testExecution.logTestStep('should load the home page');
+          await transform.navigation.navigateTo();
+          await transform.management.assertTransformListPageExists();
+
+          await transform.testExecution.logTestStep('should display the transforms table');
+          await transform.management.assertTransformsTableExists();
+
+          await transform.testExecution.logTestStep(
+            'should display the original transform in the transform list'
+          );
+          await transform.table.filterWithSearchString(transformId, 1);
+
+          await transform.testExecution.logTestStep('should start the transform');
+          await transform.table.assertTransformRowActionEnabled(transformId, 'Start', true);
+          await transform.table.clickTransformRowActionWithRetry(transformId, 'Start');
+          await transform.table.confirmStartTransform();
+          await transform.table.clearSearchString(testDataList.length);
+
+          if (testData.mode === 'continuous') {
+            await transform.testExecution.logTestStep('should display the started transform');
+            await transform.table.assertTransformRowStatusNotEql(
+              testData.originalConfig.id,
+              TRANSFORM_STATE.STOPPED
+            );
+          } else {
+            await transform.table.assertTransformRowProgress(transformId, 0);
+          }
```

Comment:
Thinking a bit more about this check, this would succeed if the continuous state was e.g. `FAILED` (same for batch after it processed some data). I think it would be good to add additional checks that the state is not `FAILED` or `ABORTING`.

Reply:
Updated here https://github.com/elastic/kibana/pull/92314/commits/58804c23861b3b5946f13c129a1a0cff0a093ab4